scalding-core/src/main/scala/com/twitter/scalding/typed/Grouped.scala

```diff
@@ -406,20 +410,20 @@ final case class UnsortedIdentityReduce[K, V1](
   override def withDescription(description: String): UnsortedIdentityReduce[K, V1] =
     copy(descriptions = descriptions :+ description)
 
-  override def filterKeys(fn: K => Boolean) =
+  override def filterKeys(fn: K => Boolean)(implicit q: Quoted) =
     UnsortedIdentityReduce(keyOrdering, mapped.filterKeys(fn), reducers, descriptions)
 
-  override def mapGroup[V3](fn: (K, Iterator[V1]) => Iterator[V3]) = {
+  override def mapGroup[V3](fn: (K, Iterator[V1]) => Iterator[V3])(implicit q: Quoted) = {
     // Only pass non-Empty iterators to subsequent functions
     IteratorMappedReduce(keyOrdering, mapped, Grouped.addEmptyGuard(fn), reducers, descriptions)
   }
 
   // It would be nice to return IdentityReduce here, but
   // the type constraints prevent it currently
-  override def mapValues[V2](fn: V1 => V2) =
+  override def mapValues[V2](fn: V1 => V2)(implicit q: Quoted) =
     UnsortedIdentityReduce(keyOrdering, mapped.mapValues(fn), reducers, descriptions)
 
-  override def sum[U >: V1](implicit sg: Semigroup[U]) = {
+  override def sum[U >: V1](implicit sg: Semigroup[U], q: Quoted) = {
```

Comment:
since semigroups emit the whole object, do we really need the quoted here? Seems like we can assume there can be no projection/filtering here.